tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,11 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
+    assert not result
+    assert item not in vendor.inventory
+    assert len(vendor.inventory) == 3
+
```

Comment:
Something that you should also explicitly test for is if the original inventory of `vendor` contains the same items as before the operation. 
```suggestion
    assert not result
    assert item not in vendor.inventory
    assert "a" in vendor.inventory
    assert "b" in vendor.inventory
    assert "c" in vendor.inventory
```
And by explictly checking these you could remove the `len(vendor.inventory) == 3` check since it will be implicitly checked by these suggested lines.

Reply:
Got it